github/issues_labels.go

```diff
@@ -21,6 +22,13 @@ func (l Label) String() string {
 	return fmt.Sprint(*l.Name)
```

Comment:
Aside from the obvious of having Label implement Stringer?  Not sure why it's using fmt.Sprint; looks like it got added in b416fdaac841e7c2404b365cf6aa7b9f1d450d93.  Should just return `*l.Name` I guess.